dpdata/plugins/vasp.py

```diff
@@ -64,7 +69,7 @@ def from_labeled_system(self, file_name, begin=0, step=1, **kwargs):
             data['energies'], \
             data['forces'], \
             tmp_virial, \
-            = dpdata.vasp.outcar.get_frames(file_name, begin=begin, step=step)
+            = dpdata.vasp.outcar.get_frames(file_name, begin=begin, step=step,ml=ml)
```

Comment:
This line does not follow [PEP-8](https://peps.python.org/pep-0008/).

Reply:
add whitespace after the comma